framework/src/node/consensus/consensus.ts

```diff
@@ -128,6 +128,7 @@ export class Consensus {
 			db: this._db,
 		} as EndpointArgs); // TODO: Remove casting in issue where commitPool is added here
 		const blockExecutor = this._createBlockExecutor();
+		this._endpoint.start();
```

Comment:
I think adding `start` in consensus and calling it from node would be more consistent